entry_tasks/14_create_contact/main.py

```diff
@@ -0,0 +1,18 @@
+if __name__ == "__main__":
+    users = {}
+    num = int(input("Enter total number of contacts to add: "))
+    for i in range(num):
+        name = input("Enter your name: ")
+        age = int(input("Enter your age: "))
+        address = input("Enter your address: ")
+        phone = input("Enter your phone: ")
+        print('>>> Contact created: ', {f'name: {name}, age: {age}, address: {address}, phone: {phone}'})
+        users[name] = phone
+        #print(f'{name}: {phone}')
+    print('Your contact book:')
+    counter = 0
+    record = {}
+    for key, value in users.items():
+        counter += 1
```

Comment:
Let's assume, loop over users will not print them in the same order as they were inserted (witch is True for the older versions of Python!)
Rework to store `i` somewhere and show it here.